qemu/tests/blockdev_stream_filter_nodename.py

```diff
@@ -0,0 +1,60 @@
+from provider import blockdev_stream_nowait
+from provider import job_utils
+
+
+class BlkdevStreamFilterNode(blockdev_stream_nowait.BlockdevStreamNowaitTest):
+    """
+    blockdev-stream filter node name test
+    """
+
+    def _init_stream_options(self):
+        super(BlkdevStreamFilterNode, self)._init_stream_options()
+        filter_node_name = self.params.get("filter_node_name")
+        if filter_node_name:
+            self._stream_options["filter-node-name"] = filter_node_name
```

Comment:
self._stream_options["filter-node-name"] = self.params['filter_node_name']
for this case is designed for filter node test, filter_node_name param is a must.
But yours is still OK for me

Reply:
@zhencliu  updated, thanks.